cmd/booster-http/server.go

```diff
@@ -155,6 +177,7 @@ func (s *HttpServer) handleByPieceCid(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// TODO: Update to parse out multiple url params
```

Comment:
Let's track in a new issue?

Reply:
Oops, I think this was an old TODO. The existing logic **can** parse out multiple params now.